src/plugins/displayLayout/CustomStringFormatter.js

```diff
@@ -0,0 +1,37 @@
+import printj from 'printj';
+
+export default class CustomStringFormatter {
+    constructor(openmct, itemFormat = null) {
+        this.openmct = openmct;
+
+        this.itemFormat = itemFormat;
+    }
+
+    format(datum, valueFormatter, valueMetadata) {
+        if (!this.itemFormat) {
+            return valueFormatter && valueFormatter.format(datum);
+        }
+
+        if (!this.itemFormat.startsWith('&')) {
+            return printj.sprintf(this.itemFormat, datum[valueMetadata.key]);
+        }
+
+        try {
+            const key = this.itemFormat.slice(1);
+            const customFormatter = this.openmct.telemetry.getFormatter(key);
+            if (!customFormatter) {
+                throw new Error('Custom Formatter not found');
+            }
+
+            return customFormatter.format(datum[valueMetadata.key]);
+        } catch (e) {
+            console.error(e);
+
+            return valueFormatter && valueFormatter.format(datum);
```

Comment:
The `CustomStringFormatter` should not be handling anything other than "custom string formatting". Regular telemetry formatting should happen in the view.

Reply:
removed regular telemetry formatting from CustomStringFormatter.

Reply:
Added Test cases.